src/index.ts

```diff
@@ -434,25 +393,19 @@ export default class ImageTool implements BlockTool {
 
   /**
    * Callback fired when Block Tune is activated
-   *
-   * @private
-   *
-   * @param {string} tuneName - tune that has been clicked
-   * @returns {void}
+   * @param tuneName - tune that has been clicked
    */
-  tuneToggled(tuneName: keyof ImageToolData): void {
+  private tuneToggled(tuneName: keyof ImageToolData): void {
     // inverse tune state
-    this.setTune(tuneName, !this._data[tuneName as keyof ImageToolData]);
+    this.setTune(tuneName, this._data[tuneName] !== undefined ? true : false);
```

Comment:
it will always set true instead of toggling

Reply:
please check now